makefile

```diff
@@ -9,6 +9,8 @@ clean:
 install:
 	npm i --no-package-lock
 	npx athloi exec -- npm i --no-package-lock
+	node scripts/link-packages-using-bower.js
+	npx	bower install
```

Comment:
I think the bower dependency will also need to be removed from each package and the `postinstall` step amended to ensure we don't double install everything 🤔

Reply:
🤔 potentially that makes sense... I will try this out

Reply:
Potentially - because none of the packages or components other than `x-podcast-launchers` seems to have a bower dependency